packages/api/types.ts

```diff
@@ -41,7 +41,7 @@ export type TSocketEndpoints = {
         response: PingResponse;
     };
     verify_email: {
-        request: VerifyEmailRequest;
+        request: Omit<VerifyEmailRequest, 'verify_email_cellxpert'> & { verify_email_cellxpert?: string };
```

Comment:
i had to add this workaround because VerifyEmailRequest type in @deriv/api-types contains a required verify_email_cellxpert field which is supposed to be non-required. this was causing failure of all checks because of this TS error thrown during the build at line 38 in packages/hooks/src/useVerifyEmail.ts: `WS.send({ verify_email: client.email, type });` saying: _"Property 'verify_email_cellxpert' is missing in type '{ verify_email: string; type: "partner_account_opening"; }' but required in type 'TSocketRequestCleaned<"verify_email">'.ts(2345)"_
index.d.ts(16488, 3)

Reply:
@maryia-binary Let's not add more patches to the FE codebase, This is a BE issue and has already been raised and fixed in this [card](https://redmine.deriv.cloud/issues/74269#Capturing_the_BTA_link_from_the_partners_registration_form_to_get_the_sub-affiliate_sign_up). Once it’s released we can update `@deriv/api-types`.

Reply:
@farzin-deriv i'll wait for the update of @deriv/api-types version then because the latest one released 2 days ago doesn't yet contain the correct type needed for the checks to pass in this PR